linkcheck/decorators.py

```diff
@@ -36,12 +36,16 @@ def h ():
 
 """
 from __future__ import print_function
+
+import codecs
```

Comment:
I don't see where this new import is getting used?

Reply:
Removed in python3_05c.